packages/driver/cypress/e2e/commands/actions/type.cy.ts

```diff
@@ -766,6 +772,7 @@ describe('src/cy/commands/actions/type - #type', () => {
       .then(() => {
         expect(cy.timeout).to.be.calledWith(5 * 8, true, 'type')
 
+        // @ts-expect-error - TODO: Get this to use the internal Keyboard types
```

Comment:
I have no idea why some specfiles don't seem to be picking up the internal types I defined for some Cypress definitions.